packages/@aws-cdk/cloudformation-include/lib/cfn-include.ts

```diff
@@ -589,7 +589,7 @@ export class CfnInclude extends core.CfnElement {
 
     // fail early for resource attributes we don't support yet
     const knownAttributes = [
-      'Type', 'Properties', 'Condition', 'DependsOn', 'Metadata', 'Version',
+      'Type', 'Properties', 'Condition', 'DependsOn', 'Metadata', 'Version', 'Description',
       'CreationPolicy', 'UpdatePolicy', 'DeletionPolicy', 'UpdateReplacePolicy',
```

Comment:
might be good to make this list alphabetical for easier reading.
don't mind if the set of policy properties follow the regular attributes.

Reply:
Sure!

```suggestion
      'Condition', 'DependsOn', 'Description', 'Metadata', 'Properties', 'Type', 'Version',
      'CreationPolicy', 'DeletionPolicy', 'UpdatePolicy', 'UpdateReplacePolicy'
```